glide.yaml

```diff
@@ -1,12 +1,12 @@
 package: github.com/giantswarm/crdstorage
 import:
+- package: github.com/giantswarm/apiextensions
+  version: release-1.8
```

Comment:
Since we are in the weird intermediate state of a version transition we have to pin this. See https://github.com/giantswarm/apiextensions/pull/35.